js/plugin/popup-dialog.ts

```diff
@@ -1,199 +1,168 @@
-import { LitElement, html, css, HTMLTemplateResult, PropertyValues } from "lit";
-import { property, query } from "lit/decorators.js";
+import { LitElement, html, css } from "lit";
+import { property, state } from "lit/decorators.js";
 import { unsafeHTML } from "lit/directives/unsafe-html.js";
 import { repeat } from "lit/directives/repeat.js";
-import {ifDefined} from 'lit/directives/if-defined.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import structuredClone from "@ungap/structured-clone";
+
 import {
   ensureArray,
   provideHass,
   selectTree,
 } from "../helpers";
 import { loadHaForm } from "../helpers";
 import { ObjectSelectorMonitor } from "../object-selector-monitor";
-import { icon } from "./types";
+import { DialogWidth, HaButtonElement, IconProps, PopupStyle } from "../types/types";
 import { BrowserModPopupParams } from "./popups";
-import structuredClone from "@ungap/structured-clone";
-
 export class BrowserModPopup extends LitElement {
-  @property() open;
```

Comment:
There is so much refactoring here irrelevant to the PR. Such can easily lead to new bugs. e.g. why change icons? And does the editor need updating? I don't know as you have not stated why icons are being updated and why? More than 'Minor design adjustments'.